extensions/Khronos/KHR_draco_mesh_compression/README.md

```diff
@@ -83,7 +83,8 @@ except `primitives`:
                             "componentType" : 5126,
                             "type" : "VEC2",
                         },
-                    ]
+                    ],
+                    "version" : "0.9.1"
```

Comment:
Does it have any conformance implications? How should that field be used by implementations?